### PR TITLE
St.nikita bondars lesson 4

### PR DESCRIPTION
**Pull request by NickBRiga**
Lesson 4 homeworks

### CODE REVIEW

File: src/main/java/students/student_Nikita_Bondars/Lesson_4/level_2/Task7.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Write first number: ");
+        int firstNumber = scanner.nextInt();
+        System.out.println("Write second number: ");
+        int secondNumber = scanner.nextInt();
+
+        if (firstNumber == secondNumber) {
+            System.out.println("Numbers are equals");
```
Comment (antdolg1) [inline src/main/java/students/student_Nikita_Bondars/Lesson_4/level_2/Task7.java:15]: equals --> equal

File: src/main/java/students/student_Nikita_Bondars/Lesson_4/level_4/Calculator.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        } else if (secondNumber == thirdNumber && secondNumber > firstNumber) {
+            return secondNumber;
+        } else if (secondNumber > thirdNumber && thirdNumber == firstNumber) {
+            return secondNumber;
+        } else if (thirdNumber > secondNumber && secondNumber == firstNumber) {
+            return thirdNumber;
+        } else if (secondNumber == firstNumber && firstNumber == thirdNumber) {
+            return firstNumber;
+        }
+        return -100500;
```
Comment (antdolg1) [inline src/main/java/students/student_Nikita_Bondars/Lesson_4/level_4/Calculator.java:60]: What is the case when this would be returned?

File: src/main/java/students/student_Nikita_Bondars/Lesson_4/level_5/LightColorDetectorTest.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        if (expectedResult.equals(actualResult)) {
+            System.out.println("--- TEST PASSED. Expected color: " + expectedResult + " ---");
+            System.out.println("======================================");
+        } else {
+            System.out.println("Test failed");
+            System.out.println("Actual result: " + actualResult);
+            System.out.println("Expected result: " + expectedResult);
+            System.out.println("======================================");
+        }
+    }
```
Comment (antdolg1) [inline src/main/java/students/student_Nikita_Bondars/Lesson_4/level_5/LightColorDetectorTest.java:119]: Please add test which will cover scenario when user entered number which is out of range.

**Review by antdolg1 — CHANGES_REQUESTED** [review]
Please check my comments.

File: src/main/java/students/student_Nikita_Bondars/Lesson_4/level_2/Task7.java (hunk 1 above)
Comment (NickBRiga) [reply src/main/java/students/student_Nikita_Bondars/Lesson_4/level_2/Task7.java:15]: +

Comment (NickBRiga) [reply src/main/java/students/student_Nikita_Bondars/Lesson_4/level_2/Task7.java:15]: done

File: src/main/java/students/student_Nikita_Bondars/Lesson_4/level_5/LightColorDetectorTest.java (hunk 3 above)
Comment (NickBRiga) [reply src/main/java/students/student_Nikita_Bondars/Lesson_4/level_5/LightColorDetectorTest.java:119]: Done

File: src/main/java/students/student_Nikita_Bondars/Lesson_4/level_4/Calculator.java (hunk 2 above)
Comment (NickBRiga) [reply src/main/java/students/student_Nikita_Bondars/Lesson_4/level_4/Calculator.java:60]: It's test for 
maxOfThreeNumbers10

Comment (antdolg1) [reply src/main/java/students/student_Nikita_Bondars/Lesson_4/level_4/Calculator.java:60]: In which scenario your method would return -100500? Do you think it is a good idea to return such a number?

**Review by antdolg1 — CHANGES_REQUESTED** [review]
I think you added lesson 6 homeworks to lesson 4 PR by mistake.
